game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -1,61 +1,51 @@
 package games.strategy.engine.lobby.client.login;
 
-import java.util.Map;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 import com.google.common.base.Strings;
 
+import lombok.Builder;
+import lombok.Getter;
+
 /**
  * Server properties.
  *
- * <p>
- * Generally there is one lobby server, but that server may move.
- * </p>
+ * <p>Generally there is one lobby server, but that server may move.
  *
- * <p>
- * To keep track of this, we always have a properties file in a constant location that points to the current lobby
- * server.
- * </p>
+ * <p>To keep track of this, we always have a properties file in a constant location that points to
+ * the current lobby server.
  *
- * <p>
- * The properties file may indicate that the server is not available using the ERROR_MESSAGE key.
- * </p>
+ * <p>The properties file may indicate that the server is not available using the ERROR_MESSAGE key.
  */
+@Builder
+@Getter
 public class LobbyServerProperties {
-  public final String host;
-  public final int port;
-  public final String serverErrorMessage;
-  public final String serverMessage;
+  /** The host address of the lobby, typically an IP address. */
+  @Nonnull private final String host;
```

Comment:
Another consideration for the forthcoming formatting issue...  The formatter will also force `@Nonnull` here to be on its own line because it considers it a field annotation.  Regardless of whether or not field annotations should be on separate lines, `@Nonnull` and `@Nullable` in this context are really type annotations, so they technically should be next to the type they are annotating (i.e. `private final @Nonnull String host;`).

(Although, the `javax.annotation` annotations pre-date Java 8, and thus are not valid type annotations.  We might want to consider switching to the `org.checkerframework` nullity annotations for this reason.  I believe the latest versions of Guava have switched to this library.)